docker/docker-entrypoint.sh

```diff
@@ -1,28 +1,13 @@
 #!/bin/sh
 
-# Check if the "characters" directory is empty
-if [ -z "$(ls -A /home/node/app/config/characters)" ]; then
-  echo "Characters directory is empty. Copying default characters."
-  mv /home/node/app/public/characters.default /home/node/app/config/characters
-fi
-
-# Check if the "chats" directory is empty
-if [ -z "$(ls -A /home/node/app/config/chats)" ]; then
-  echo "Chats directory is empty. Copying default chats."
-  mv /home/node/app/public/chats.default /home/node/app/config/chats/
-fi
-
-# Check if the "User Avatars" directory is empty
-if [ -z "$(ls -A '/home/node/app/config/User Avatars')" ]; then
-  echo "User Avatars directory is empty. Copying default user avatars."
-  mv /home/node/app/public/User\ Avatars.default /home/node/app/config/User\ Avatars/
-fi
-
-# Check if the "settings.json" file is not empty
-if [ ! -s "/home/node/app/config/settings.json" ]; then
-  echo "Settings file does not exist. Copying default settings."
-  mv /home/node/app/public/settings.json.default /home/node/app/config/settings.json
-fi
+# Initialize missing user files
+IFS="," RESOURCES="characters,chats,User Avatars,settings.json"
+for R in $RESOURCES; do
+  if [ ! -e "config/$R" ]; then
+    echo "Resource not found, copying from defaults: $R"
+    cp -r "public/$R.default" "config/$R"
+  fi
+done
```

Comment:
Before this PR:
* `/home/node/app/config/characters` does not exist – it is initialized from the defaults.
* `/home/node/app/config/characters` exists and is not empty – nothing happens.
* `/home/node/app/config/characters` exists but is empty – an attempt is made to initialize it from defaults, but the end result is that the defaults end up in a deeper directory – `/home/node/app/config/characters/characters.defaults`.
* Initialization process moves the files rather than copies them, so it works at most once for each directory. If you break/delete the configuration, you have to create a new container.

After this PR:
* `/home/node/app/config/characters` does not exist – it is initialized from the defaults.
* `/home/node/app/config/characters` exists, empty or not – nothing happens. Someone may want it empty for some reason, let's not assume. To restore the default configuration, just delete the directory entirely.
* Initialization process copies the files, so it is possible to e.g. delete `settings.json` and restart the container to restore the default settings.